src/parsers/manifestjson.js

```diff
@@ -478,6 +480,23 @@ export default class ManifestJSONParser extends JSONParser {
         });
       }
     }
+
+    if (this.parsedJSON.name && typeof this.parsedJSON.name === 'string') {
+      const messageNameRegexp = new RegExp(MANFIEST_MESSAGE_NAME_REGEXP, 'g');
+      const match = messageNameRegexp.exec(this.parsedJSON.name);
```

Comment:
Nit: No need to declare a `const` (or variable) if the value expression is simple and only used once. You can move it inside the `if`.